### PR TITLE
Add even more coding standard fixers

### PR DESCRIPTION
**Pull request by pamil**
| Q               | A
| --------------- | ---
| Bug fix?        | no |
| New feature?    | no |
| BC breaks?      | no |
| Related tickets | continuation of #8634 |
| License         | MIT |

### CODE REVIEW

File: src/Sylius/Behat/Context/Cli/InstallerContext.php [hunk 1]
```diff
@@ -150,7 +150,7 @@ public function iProvideFullAdministratorData()
      *
      * @return resource
      */
-    protected function getInputStream($input)
+    private function getInputStream($input)
```
Comment (pamil) [inline src/Sylius/Behat/Context/Cli/InstallerContext.php]: No BC break here, changing method visibility from protected to private happens only for final classes.

File: src/Sylius/Behat/Context/Ui/Admin/ManagingProductVariantsContext.php [hunk 2]
```diff
@@ -158,7 +158,7 @@ public function iEnableItsTracking()
      */
     public function iSetItsPriceTo($price = null, $channelName = null)
     {
-        $this->createPage->specifyPrice($price, (null === $channelName) ? $this->sharedStorage->get('channel') :$channelName);
+        $this->createPage->specifyPrice($price, (null === $channelName) ? $this->sharedStorage->get('channel') : $channelName);
```
Comment (pamil) [inline src/Sylius/Behat/Context/Ui/Admin/ManagingProductVariantsContext.php]: `$channelName ?? $this->sharedStorage->get('channel')` can be used here

File: src/Sylius/Behat/Page/Page.php [hunk 3]
```diff
@@ -146,7 +146,7 @@ protected function getParameter($name)
     /**
      * Defines elements by returning an array with items being:
      *  - :elementName => :cssLocator
-     *  - :elementName => [:selectorType => :locator]
+     *  - :elementName => [:selectorType => :locator].
```
Comment (pamil) [inline src/Sylius/Behat/Page/Page.php]: This check isn't as useful as I thought.

File: src/Sylius/Behat/Page/Shop/Cart/SummaryPage.php [hunk 4]
```diff
@@ -200,7 +200,7 @@ public function hasItemWithCode($code)
     }
 
     /**
-     * {@inheritdoc]
+     * {@inheritdoc].
```
Comment (pamil) [inline src/Sylius/Behat/Page/Shop/Cart/SummaryPage.php]: Needs to be changed to `{@inheritdoc}`

File: src/Sylius/Bundle/CoreBundle/Validator/Constraints/HasAllVariantPricesDefinedValidator.php [hunk 5]
```diff
@@ -39,7 +39,7 @@ public function validate($product, Constraint $constraint): void
 
         $channels = $product->getChannels();
 
-        /** @var ProductVariantInterface $variant */
+        /* @var ProductVariantInterface $variant */
```
Comment (pamil) [inline src/Sylius/Bundle/CoreBundle/Validator/Constraints/HasAllVariantPricesDefinedValidator.php]: This one's good though

File: src/Sylius/Bundle/FixturesBundle/Loader/SuiteLoader.php [hunk 6]
```diff
@@ -40,7 +40,7 @@ public function __construct(FixtureLoaderInterface $fixtureLoader)
     public function load(SuiteInterface $suite): void
     {
         /**
-         * @var FixtureInterface $fixture
+         * @var FixtureInterface
```
Comment (pamil) [inline src/Sylius/Bundle/FixturesBundle/Loader/SuiteLoader.php]: :(

File: src/Sylius/Component/Core/Model/OrderInterface.php [hunk 7]
```diff
@@ -26,13 +26,7 @@
...
  * @author Paweł Jędrzejewski <pawel@sylius.org>
  */
-interface OrderInterface extends
-    BaseOrderInterface,
-    PaymentsSubjectInterface,
-    CountablePromotionSubjectInterface,
-    PromotionCouponAwarePromotionSubjectInterface,
-    CustomerAwareInterface,
-    ChannelAwareInterface
+interface OrderInterface extends BaseOrderInterface, PaymentsSubjectInterface, CountablePromotionSubjectInterface, PromotionCouponAwarePromotionSubjectInterface, CustomerAwareInterface, ChannelAwareInterface
```
Comment (GSadee) [inline src/Sylius/Component/Core/Model/OrderInterface.php]: Why? 😱

File: src/Sylius/Bundle/FixturesBundle/Loader/SuiteLoader.php [hunk 8]
```diff
@@ -39,10 +39,8 @@ public function __construct(FixtureLoaderInterface $fixtureLoader)
      */
     public function load(SuiteInterface $suite): void
     {
-        /**
-         * @var FixtureInterface
-         * @var array $fixtureOptions
-         */
+        /** @var FixtureInterface $fixture */
```
Comment (GSadee) [inline src/Sylius/Bundle/FixturesBundle/Loader/SuiteLoader.php]: Shouldn't be `/*` instead of `/**`?

Comment (pamil) [reply src/Sylius/Bundle/FixturesBundle/Loader/SuiteLoader.php]: It's a PHPdoc comment

File: src/Sylius/Component/Core/Model/OrderInterface.php (hunk 7 above)
Comment (pamil) [reply src/Sylius/Component/Core/Model/OrderInterface.php]: Something has gone wrong :)

**Comment by GSadee** [comment]
Thanks Kamil :1st_place_medal:

File: src/Sylius/Behat/Context/Cli/InstallerContext.php (hunk 1 above)
Comment (pjedrzejewski) [reply src/Sylius/Behat/Context/Cli/InstallerContext.php]: :)

**Pull request merged**